src/components/tabs/tabs.js

```diff
@@ -296,17 +306,21 @@ export default Vue.extend({
       this.updateTabs()
       // Observe child changes so we can update list of tabs
       this.setObserver(true)
+      // Flag we are now mounted and to switch to DOM for tab probing
+      this.isMounted = true
     })
   },
   deactivated() /* istanbul ignore next */ {
     this.setObserver(false)
+    this.isMounted = false
   },
   activated() /* istanbul ignore next */ {
     let tabIdx = parseInt(this.value, 10)
     this.currentTab = isNaN(tabIdx) ? -1 : tabIdx
     this.$nextTick(() => {
       this.updateTabs()
       this.setObserver(true)
+      this.isMounted = true
```

Comment:
Do we still need the `this.updateTabs()` before the `this.setObserver(true)` call since `this.isMounted = true` updates the tabs afterwards anyway?

Reply:
Same as below...